### ISSUE TITLE
IDs stored as int instead of long

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lukaskabrt/spatiallite-net#2]
```
What steps will reproduce the problem?
  1. Read an OSM file (my test was against a .PBF file) that has an entity with an ID > 2^31


What is the expected output? What do you see instead?
  IEntityInfo.ID should be a long (Int64) - it is currently int (Int32).
  Entities with ID > 2^31 result in overflow errors (i.e. negative IDs). 


What version of the product are you using? On what operating system?
  SpatialLITE_1.1.0 on Windows 7x64, .NET 4.5


Please provide any additional information below.
  Negative Int64 IDs are often used locally to indicate additions that have not been submitted to the OSM databases.

```

Original issue reported on code.google.com by `jj.jabba...@gmail.com` on 2 Jul 2013 at 2:42

**Comment by archanox** [comment]
Just came here to report the same issue.

Specifically trying to read in an osm node id of 3199613538

using the xml parser from the url https://www.openstreetmap.org/api/0.6/node/3199613538

**Comment by lukaskabrt** [comment]
Fixed in the 2.0.0 RC
